### DISCUSSION TITLE
Shutdown feature

### DISCUSSION THREAD

**Discussion by SwineFlew931** [Donkie/Spoolman#457] [Ideas]
I run Spoolman as a standalone server on a pi to serve several printers. Spoolman is the only thing on it other than the pi OS. Would it be possible to incorporate a Shutdown button? Perhaps make it an option to toggle on or off in the Settings/General area, for those that want to shutdown the pi 'gracefully' without opening a terminal and logging in to stop the service and shutdown the pi?
Or is there something I'm missing?

Thanks!

**Comment by Donkie** [comment]
If you just shutdown your pi, Spoolman should exit cleanly. Integrating a shutdown button in Spoolman is way outside the scope of what it's meant to do.
